packages/component/src/components/page-list/operation-menu-items/export.tsx

```diff
@@ -89,33 +101,57 @@ const ExportToHtmlMenuItem = ({
   );
 };
 
-// const ExportToPngMenuItem = ({
-//   onSelect,
-// }: CommonMenuItemProps<{ type: 'png' }>) => {
-//   const t = useAFFiNEI18N();
-//   const contentParserRef = useRef<ContentParser>();
-//   return (
-//     <>
-//       {globalThis.currentEditor!.mode === 'page' && (
-//         <MenuItem
-//           data-testid="export-to-png"
-//           onClick={() => {
-//             if (!contentParserRef.current) {
-//               contentParserRef.current = new ContentParser(
-//                 globalThis.currentEditor!.page
-//               );
-//             }
-//             contentParserRef.current.exportPng();
-//             onSelect?.({ type: 'png' });
-//           }}
-//           icon={<ExportToPngIcon />}
-//         >
-//           {t['Export to PNG']()}
-//         </MenuItem>
-//       )}
-//     </>
-//   );
-// };
+const ExportToPngMenuItem = ({
+  onSelect,
+}: CommonMenuItemProps<{ type: 'png' }>) => {
+  const t = useAFFiNEI18N();
+  const contentParserRef = useRef<ContentParser>();
+  const { currentEditor } = globalThis;
+  const currentWorkspaceId = useAtomValue(rootCurrentWorkspaceIdAtom);
+  const currentPageId = useAtomValue(rootCurrentPageIdAtom);
+  const onClickDownloadPNG = useCallback(() => {
+    if (!currentEditor) {
+      return;
+    }
+    const contentParser =
+      contentParserRef.current ??
+      (contentParserRef.current = new ContentParser(currentEditor.page));
+
+    window.apis?.export
+      .savePngFileAs(
+        currentWorkspaceId || '',
+        currentPageId || '',
+        (currentEditor.page.root as PageBlockModel).title.toString(),
+        currentEditor.mode
+      )
+      .then(result => {
+        if (result !== undefined) {
+          return;
+        }
+        return contentParser.exportPng();
```

Comment:
Does this mean this feature can work without creating a new window? If if it is the case, I think we can just do it purely in here instead of creating a window?